geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/processors/datastore/PutGeoMesaFileSystem.scala

```diff
@@ -6,15 +6,15 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  ***********************************************************************/
 
-package org.geomesa.nifi.processors.datastore
+package org.geomesa.nifi.processors
+package datastore
```

Comment:
Are you liking this way of specifying the package?

Reply:
it brings the parent package object in scope, which is nice